src/main/java/com/linkedin/kmf/services/ConsumerFactoryImpl.java

```diff
@@ -34,9 +37,16 @@
       new String[] {ConsumeServiceConfig.BOOTSTRAP_SERVERS_CONFIG, ConsumeServiceConfig.ZOOKEEPER_CONNECT_CONFIG};
   private int _latencySlaMs;
   private static AdminClient adminClient;
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConsumerFactoryImpl.class);
```

Comment:
I think the Logger variable name is `LOG` instead of `LOGGER` through out this codebase. In most other codebases, the logger variable name is `LOG` instead of `LOGGER` as far as I have seen

Reply:
Sure.